### PR TITLE
Fix ktx texture orientation in sandbox

### PR DESCRIPTION
**Pull request by bghgary**
See https://github.com/KhronosGroup/KTX-Software/pull/959 for full context.

The gist of it is that KTX textures are y inverted when previewing in the sandbox. The reason why is because [passing in `invertY` of true doesn't actually work for compressed textures](https://forum.babylonjs.com/t/make-inverty-flag-more-consistent-across-different-texture-loading-scenarios/52751). I fixed it by updating the UVs of the preview plane to be OpenGL conventions so that all textures behave the same.

### CODE REVIEW

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Visualization tests for WebGPU (Experimental)

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15853/merge/testResults/webgpuplaywright/index.html

**Comment by bjsplat** [comment]
WebGL2 visualization test reporter:

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15853/merge/testResults/webgl2playwright/index.html

**Comment by bjsplat** [comment]
Visualization tests for WebGPU (Experimental)

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15853/merge/testResults/webgpuplaywright/index.html

**Comment by bjsplat** [comment]
WebGL2 visualization test reporter:

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15853/merge/testResults/webgl2playwright/index.html

File: packages/dev/core/src/Compat/index.ts [hunk 1]
```diff
@@ -1 +1 @@
-export { CompatibilityOptions } from "./compatibilityOptions";
+export * from "./compatibilityOptions";
```
Comment (bghgary) [inline packages/dev/core/src/Compat/index.ts:1]: I had to make this change or I get this error from running the sandbox locally:
image

**Review by sebavan — APPROVED** [review]

**Pull request merged**